### DISCUSSION TITLE
RSX parser suggestion: `syn-rsx`

### DISCUSSION THREAD

**Discussion by stoically** [DioxusLabs/dioxus#620] [Ideas]
Hey, your RSX parser is probably feature complete and there's no interest in changing it – but in case you have to expand functionality: I just wanted to leave an hint for my [syn-rsx](https://github.com/stoically/syn-rsx) crate here. I think it'd be great to combine efforts for that basic building block, and others like [sauron](https://github.com/ivanceras/sauron) and [leptos](https://github.com/gbj/leptos) are successfully using it.

If you happen to give it a look and something is missing for your use case, I'd be interested in the details. Currently there's also work happening on supporting unquoted text behind a nightly feature flag: https://github.com/stoically/syn-rsx/issues/2

Sorry for the shameless plug and obviously feel free to close/delete this discussion.

**Comment by ealmloff** [comment]
We also have a [html-like macro](https://github.com/DioxusLabs/dioxus-html-macro/blob/main/Cargo.toml). It might be better to use syn-rsx as a parser for that crate instead of a custom solution

**Reply by stoically** [reply]
Somehow I thought that crate is outdated, hence why I closed https://github.com/DioxusLabs/dioxus-html-macro/issues/1. But if it's still up to date, then it might indeed make sense!
